tools/devtool

```diff
@@ -641,6 +641,9 @@ cmd_test() {
                 shift
                 local cpuset_mems="$1"
                 ;;
+            "--performance")
+                local performance_tweaks=1;
+                ;;
```

Comment:
Should we add this flag to the help message as well?

Reply:
I'd prefer not to, I'd really like to avoid this being used outside of the CI

Reply:
ok, agreed